frontend/public/components/resource-quota.jsx

```diff
@@ -42,10 +47,18 @@ const clusterResourceQuotaMenuActions = [
 
 const isClusterQuota = (quota) => !quota.metadata.namespace;
 
-const quotaKind = (quota) =>
-  isClusterQuota(quota)
-    ? referenceForModel(ClusterResourceQuotaModel)
-    : referenceForModel(ResourceQuotaModel);
+const resourceQuotaReference = referenceForModel(ResourceQuotaModel);
+const clusterQuotaReference = referenceForModel(ClusterResourceQuotaModel);
+const appliedClusterQuotaReference = referenceForModel(AppliedClusterResourceQuotaModel);
+
+const quotaKind = (quota, canListClusterQuota) => {
+  if (quota.metadata.namespace) {
+    return resourceQuotaReference;
+  }
+
+  return canListClusterQuota ? clusterQuotaReference : appliedClusterQuotaReference;
```

Comment:
So I think we probably always want to use `AppliedClusterResourceQuota` regardless of access. That way we can differentiate between used for project and total used, and we can even skip the access check.